Userland/Utilities/mv.cpp

```diff
@@ -19,15 +19,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     TRY(Core::System::pledge("stdio rpath wpath cpath fattr"));
 
-    // NOTE: The "force" option is a dummy for now, it's just here to silence scripts that use "mv -f"
-    //       In the future, it might be used to cancel out an "-i" interactive option.
```

Comment:
I think this comment can stay, `force` still doesn't really do anything right?

Reply:
I think the comment would be misleading if we kept it. Force now cancels out the no-clobber flag, just like it will eventually cancel out the interactive flag.

Reply:
Alright that's fine, we can keep it then